### DISCUSSION TITLE
Dynamic Range Compander settings

### DISCUSSION THREAD

**Discussion by Npa972** [Audio4Linux/JDSP4Linux#193] [Q&A] [answered]
Hi everybody, can someone explain to me how the compander works ?

I've played with it but I don't really see any noticeable differences.

In any case thank you to the developers for this software !

Version : jsdp-2.7.0-git
OS archlinux
sound server : pipewire

**Comment by james34602** [comment, marked as answer]
Dynamic Range Compander is a auto-threshold dynamic range boosting/reducing system.

You should hear good amount of difference for signal with some transient sound in it, like drums
